### ISSUE TITLE
🐛 lint/a11y/useAltText: False positive for Vue dynamic :alt binding in .vue files

### ISSUE THREAD

**Issue by SchahinRohani** [biomejs/biome#9349]
### Environment information

<details>

```bash

$ biome rage
CLI:
  Version:                      2.4.4
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v25.7.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.3.10

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true
  HTML full support enabled:    true

Workspace:
  Open Documents:               0

```

</details>

### What happened?

`useAltText` triggers a false positive in Vue SFCs when the `alt` attribute
is bound dynamically via `:alt` (i.e. `v-bind:alt`).
```vue
<template>
  <img :src="image.src" :alt="image.description" />
</template>
```

Playground reproduction: [Playground](https://biomejs.dev/playground/?code=PAB0AGUAbQBwAGwAYQB0AGUAPgAKACAAIAA8AGkAbQBnACAAOgBzAHIAYwA9ACIAaQBtAGEAZwBlAC4AcwByAGMAIgAgADoAYQBsAHQAPQAiAGkAbQBhAGcAZQAuAGQAZQBzAGMAcgBpAHAAdABpAG8AbgAiACAALwA%2BAAoAPAAvAHQAZQBtAHAAbABhAHQAZQA%2BAA%3D%3D&language=vue)

### Expected result

`:alt="..."` should be treated as a valid `alt` attribute, equivalent to
`alt="..."`, and should suppress the `useAltText` diagnostic.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
Please include a playground link

**Comment by SchahinRohani** [comment]
Used the playground for the first time, hope the reproduction is enough (set the language to vue and added the code)
Also added the playground reproduction to the issue description.

Let me know if this is enough @dyc3
